roles/StackStorm.mongodb/vars/debian.yml

```diff
@@ -1,6 +1,6 @@
 # Use the following URL to find the key: https://www.mongodb.org/static/pgp/server-{{ mongodb_major_minor_version }}.asc
 mongodb_apt_keys:
-  "3.2": "42F3E95A2C4F08279C4960ADD68FA50FEA312927"
-  "3.4": "0C49F3730359A14518585931BC711F9BA15703C6"
   "3.6": "2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5"
   "4.0": "9DA31620334BD75D9DCB49F368818C72E52529D4"
+  "4.2": "3F75C80EE56C41F4E49515B62ACD4032058F8B6B"
```

Comment:
I think as we haven't yet tested with 4.2, then I'd be tempted to only add 4.4 for now.
Add 4.2 when we confirm if we want to support it as part of 3.6

Reply:
Ok, fine for me. I removed it.